app/components/account/FeatureAccountSection.tsx

```diff
@@ -20,42 +27,157 @@ type Props = Readonly<{
 }>;
 
 const FeatureCard = ({ account }: Props) => {
+    const { cluster } = useCluster();
     const feature = parseFeatureAccount(account);
-    let activatedAt;
+    const featureInfo = useMemo(() => getFeatureInfo(feature.address), [feature.address]);
+
+    let activatedAtSlot;
+    let clusterActivation;
+    let simdLink;
     if (feature.activatedAt) {
-        activatedAt = (
+        activatedAtSlot = (
             <tr>
-                <td>Activated At Slot</td>
+                <td className="text-nowrap">Activated At Slot</td>
                 <td className="text-lg-end">
                     <code>{feature.activatedAt}</code>
                 </td>
             </tr>
         );
     }
+    if (feature.activatedAt && featureInfo) {
+        clusterActivation = (
+            <tr>
+                <td className="text-nowrap">Cluster Activation</td>
+                <td className="text-lg-end">
+                    <ClusterActivationEpochAtCluster featureInfo={featureInfo} cluster={cluster} />
+                </td>
+            </tr>
+        );
+        simdLink = (
+            <tr>
+                <td>SIMD</td>
+                <td className="text-lg-end">
+                    {featureInfo.simd && featureInfo.simd_link ? (
+                        <a
+                            href={featureInfo.simd_link}
+                            target="_blank"
+                            rel="noopener noreferrer"
+                            className=""
+                        >
+                            See SIMD {featureInfo.simd}
+                        </a>
```

Comment:
nit: remove "See SIMD" and just have "SIMD {featureInfo.simd}" and the little logo that tells people it will open a new tab/window

Reply:
Updated

Reply:
<img width="700" alt="image" src="https://github.com/user-attachments/assets/3e4cf7f1-b223-40e3-9460-7d63baa21d5e" />